### ISSUE TITLE
swgui for Windows deprecated

### ISSUE THREAD

**Issue by MK-3PP** [SoftwareNetwork/sw#120]
Hello, 

after fixing the C++23  [issue](https://github.com/SoftwareNetwork/sw/issues/115#issuecomment-1715607788) with older Visual C++ compilers, cmake still did not like me and I tried building tesseract with swgui for windows. Unfortunately, the builds on https://software-network.org/client/ are from December 2022.

And thus I get an error:

> [140/140] D:/Tools/sw/.sw/cfg/258933/loc.sw.self.d8df57-0.0.30.dll
Exception in file D:/dev/cppan2/client2/src/sw/driver/module.cpp:74, function Module: Bad config ABI version. Module ABI (31) is greater than binary ABI (30). Update your sw binary. Will rebuild on the next run.

I can build it using the command line client from June 2023, but it is cumbersome to configure.

Bottom line: Maybe update the binary clients more frequently :-)

**Comment by egorpugin** [comment]
Hi,

I've updated swgui. Try it now.

Be sure you are downloading the right file with arch suffix.
https://software-network.org/client/swgui-master-windows_x86_64-client.zip

**Comment by MK-3PP** [comment]
Sorry, already made it using just cmake. Appreciating the update though.
